Recording/Stages/SendFountainInfo.asm

```diff
@@ -0,0 +1,52 @@
+################################################################################
+# Address: 801cc998
+################################################################################
+.include "Common/Common.s"
+.include "Recording/Recording.s"
+
+################################################################################
+# Routine: SendFountainInfo
+# ------------------------------------------------------------------------------
+# Sends Fount of Dreams platform heights when they change.
+################################################################################
+
+.set REG_Buffer,4
```

Comment:
This can be slightly sketchy to do because at least usually I expect registers 4, 5, etc to be available for short term operations. Might be better to use r12 and r11 instead? And obviously when you do this you always have to be cautious that you never add any function calls and expect the values of these registers to persist.